src/core/libmaven/eiclogic.h

```diff
@@ -34,14 +34,17 @@ class EICLogic {
 								int intensityWeight,
 								int deltaRTWeight);
 
-	void groupPeaks(float eic_smoothingWindow,
-					float grouping_maxRtWindow,
-					double minQuality,
-					double distXWeight,
-					double distYWeight,
-					double overlapWeight,
-					bool useOverlap,
-					double minSignalBaselineDifference);
+    void groupPeaks(float eic_smoothingWindow,
+                    Compound* compound,
+                    float grouping_maxRtWindow,
+                    double minQuality,
+                    double distXWeight,
+                    double distYWeight,
+                    double overlapWeight,
+                    bool useOverlap,
+                    double minSignalBaselineDifference,
+                    float productPpmTolerance,
+                    string scoringAlgo);
```

Comment:
Should have included this file in the previous commit or the part from `EicWidget::groupPeaks` of last commit should have been included in this commit. I was wondering how this method was even compiling with two missing parameters.

Reply:
That was a staging error on my part. Tried to break the whole change into different commits. Must have missed that.